packages/browser-wallet/src/popup/shared/PageHeader/PageHeader.tsx

```diff
@@ -10,9 +10,18 @@ import Button from '../Button';
 type BaseProps = ClassName & {
     // eslint-disable-next-line react/no-unused-prop-types
     children: string;
-    // eslint-disable-next-line react/no-unused-prop-types
-    canGoBack?: boolean;
-};
+} & (
+        | {
+              // eslint-disable-next-line react/no-unused-prop-types
```

Comment:
Maybe just throw in a 
```suggestion
              /* eslint-disable react/no-unused-prop-types */
```
somewhere :upside_down_face: